src/content/docs/telemetry-data-platform/understand-data/event-data/events-reported-browser-monitoring.mdx

```diff
@@ -73,11 +73,11 @@ redirects:
 
     <tr>
       <td>
-        [`AjaxRequest` (SPA](/attribute-dictionary/?attribute_name=&events_tids%5B%5D=8302&event=AjaxRequest))
+        [`AjaxRequest`](/attribute-dictionary/?attribute_name=&events_tids%5B%5D=8302&event=AjaxRequest)
       </td>
 
       <td>
-        An `AjaxRequest` event is created automatically when an Ajax request occurs during a `BrowserInteraction` event. The event attributes track geographic and browser info.
+        An `AjaxRequest` event is created any time an Ajax call is made. This can happen either automatically when an Ajax request occurs during a `BrowserInteraction` event or during any stand-alone Ajax request. The event attributes track geographic and browser info.
```

Comment:
I would take out the second sentence and instead say something like -
"When an ajax call happens during a BrowserInteraction event, it is decorated with additional attributes related to the browser interaction"